var/spack/repos/builtin/packages/ratel/package.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Ratel(MakefilePackage, CudaPackage, ROCmPackage):
+    """Extensible, performance-portable solid mechanics with libCEED and PETSc"""
+
+    homepage = "https://ratel.micromorph.org"
+    git = "https://gitlab.com/micromorph/ratel.git"
+
+    maintainers = ['jedbrown', 'jeremylt']
+
+    # development version
+    version('develop', branch='main')
+    depends_on('libceed@develop', when='@develop')
+    depends_on('petsc@main', when='@develop')
+
+    # version 0.1.1
+    version('0.1.1', tag='v0.1.1')
+    depends_on('libceed@0.10.1', when='@0.1.1')
+    depends_on('petsc@3.17.0', when='@0.1.1')
+
+    # Note: '+cuda' and 'cuda_arch' variants are added by the CudaPackage
+    #       '+rocm' and 'amdgpu_target' variants are added by the ROCmPackage
+    # But we need to sync cuda/rocm with libCEED and PETSc
+    for sm_ in CudaPackage.cuda_arch_values:
+        depends_on('libceed+cuda cuda_arch={0}'.format(sm_),
+                   when='+cuda cuda_arch={0}'.format(sm_))
+        depends_on('petsc+cuda cuda_arch={0}'.format(sm_),
+                   when='+cuda cuda_arch={0}'.format(sm_))
+    for gfx in ROCmPackage.amdgpu_targets:
+        depends_on('libceed+rocm amdgpu_target={0}'.format(gfx),
+                   when='+rocm amdgpu_target={0}'.format(gfx))
+        depends_on('petsc+rocm amdgpu_target={0}'.format(gfx),
+                   when='+rocm amdgpu_target={0}'.format(gfx))
+    # Kokkos required for AMD GPUs
+    depends_on('petsc+kokkos', when='+rocm')
+
+    def configure(self, spec, prefix):
+        options = [
+            'CEED_DIR=%s' % spec['libceed'].prefix,
+            'PETSC_DIR=%s' % spec['petsc'].prefix,
+        ]
+        make('configure', *options)
```

Comment:
When is this function called?

Reply:
drat, I tried to mirror your changes in the libceed package but this slipped through the cracks. This should proceed any build/install/test via make

Reply:
I think I fixed it in the most recent commit